com.unity.render-pipelines.high-definition/Documentation~/whats-new-10-0.md

```diff
@@ -283,6 +283,11 @@ HDRP's ray-traced reflection (RTR) solution now support transparent materials. T
 
 Ray tracing now supports VR. However, since ray tracing is resource intensive and VR amplifies this, the performance is very slow.
 
+### Render Graph
+
+HDRP now internally uses a Render Graph system. This has no impact on features visible to the users but it should improve overall memory usage significantly. In the new HDRP template, the GPU memory usage decreased by 25%.
```

Comment:
HDRP now internally uses a Render Graph system. This has no impact on features available to you and it should improve overall memory usage significantly. In the new HDRP template, GPU memory usage decreased by 25%.